src/components/UI/Button/Button.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Button.scss';
+
+const Button = ({
+  name,
+  clickHandler,
+  className,
+  alignRight,
+}) => (
+  <button
+    onClick={clickHandler}
+    className={[styles.button, className, alignRight ? styles.right : ''].join(' ')}
+    type="button">
```

Comment:
place closing  > bracket on new line to increase readability

Reply:
ok